website/cue/reference/components/sinks/base/databend.cue

```diff
@@ -106,6 +106,30 @@ base: components: sinks: databend: configuration: {
 			}
 		}
 	}
+	compression: {
+		description: """
+			Compression configuration.
+
+			All compression algorithms use the default compression level unless otherwise specified.
+			"""
+		required: false
+		type: string: {
+			default: "gzip"
+			enum: {
+				gzip: """
+					[Gzip][gzip] compression.
```

Comment:
As far as I understand, Databend supports more compression algorithms: https://databend.rs/doc/load-data/#supported-file-formats

Is it true? If yes, would you like to add support for more compression algorithms, supported by databend?

Reply:
That's true.
Since there are only `Gzip` and `Zlib` in `sink::util:buffer::compression::Compression`, we could add more compression format support for vector later.

Reply:
Well, I do not recommend using `sink::util:buffer::compression::Compression` for sinks-specific compression settings since every sink has own set of supported compression algorithms. `sink::util:buffer::compression::Compression` is too generic.

I advise creating a Databend-specific `Compression` enum, filling it with supported by Databend algorithms, and then using it for compression purposes.

However, I see that here you are bounded by the trait so you cannot use this approach :( Hopefully, later we will extend `sink::util:buffer::compression::Compression` with more algorithms...